index.html

```diff
@@ -95,16 +95,16 @@ <h2 class="about">About myself</h2>
       <div class="formsection">
         <h3 class="styleh3">CONTACT US</h3>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        <form action="" class="formstyle">
-          <input type="text" placeholder="Text input">
-          <input type="email" placeholder="email">
-          <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/moqrojze" method="POST" class="formstyle">
+          <input type="text" placeholder="Text input" maxlength="30" required>
+          <input type="email" placeholder="email" required>
+          <textarea name="text" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+          <button type="submit" class="button_submit"><span>Submit</span></button>
```

Comment:
- [ ] As the previous reviewer mentioned you should add interactivity to the button, with the same colors as in the Figma design. Kindly fix your color choice so it's like Figma 

- Your button on hover:
![Screenshot_20220705_202327](https://user-images.githubusercontent.com/25227022/177401303-3c1265e3-e6e2-401e-90d0-d1a713ad02b5.png)

- Your button on active state:
![Screenshot_20220705_202808](https://user-images.githubusercontent.com/25227022/177401440-42972025-d803-456f-9704-3d20fe32e433.png)

- Figma design:
![Screenshot_20220705_202509](https://user-images.githubusercontent.com/25227022/177401471-538cff4b-b578-4466-94cb-1dc52a57b081.png)

Reply:
Ok, l literally copy the hex color from the Figma.